charts/signoz/values.yaml

```diff
@@ -1404,6 +1404,41 @@ otelCollector:
           filesystem: {}
           network: {}
     processors:
+      # default parsing of logs
+      logstransform/default:
```

Comment:
can we use `logstransform/internal` instead?

Reply:
Changed.